src/lib.rs

```diff
@@ -61,12 +64,20 @@ pub fn get_args() -> DCPResult<Config> {
                 .takes_value(false)
                 .short("w")
                 .long("write-to-stdout")
+        ).arg(
+            Arg::with_name("log_level")
+                .value_name("LOG_LEVEL")
+                .help("What level of logs to output. Accepts: [info, debug, trace, error, warn]")
+                .short("l")
+                .long("log_level")
+                .default_value("info")
```

Comment:
I actually would like the default level to be debug. I think having the pull output is helpful -- it shows the user the tool is working. Other tools like opm render as @timflannagan mentioned look like they're hanging when you run the command, since no logs are emitted.